resources/public/api/conf/1.0/schemas/selfAssessment/retrieveTaxCalc/retrieve_incomeTaxAndNics_response.json

```
@@ -1595,19 +1595,19 @@
           "properties": {
             "savings": {
               "type": "number",
-              "description": "Tax deducted at source from interest received from UK banks and building societies. This value must be between 0 and 99999999999. No decimals.",
+              "description": "Tax deducted at source from interest received from UK banks and building societies. This value must be between 0 and 99999999999.99 up to 2 decimal places.",
               "multipleOf": 1,
```

This should be 
```suggestion
              "multipleOf": 0.01,
```
for decimal fields